omise/omise.php

```diff
@@ -0,0 +1,44 @@
+<?php
+if (! defined('_PS_VERSION_'))
+    exit();
+
+class Omise extends PaymentModule
+{
+
+    /**
+     * The name that will be display to the user at the back-end.
+     *
+     * @var string
+     */
+    const MODULE_DISPLAY_NAME = 'Omise';
+
+    /**
+     * The name that used to referece in the program.
+     *
+     * @var string
+     */
+    const MODULE_NAME = 'omise';
+
+    /**
+     * The version of the module.
+     *
+     * @var string
+     */
+    const MODULE_VERSION = '1.6.0.0';
+
+    public function __construct()
+    {
+        $this->name = self::MODULE_NAME;
```

Comment:
@nimid I don't think so. those standard didn't say doesn't mean we should not do it (unless it gonna break something or some rules that those standard had set).

To support code readability, I think alignment of assignments is important.
But anyway, we can say 👆 style could be broken when the length of the variable makes another surrounding variable looks strange.

As for the automated tool, you can search for "alignment of assignments" keyword,
I did and found tools like that for `PHPStrom`, `Sublime3` & `NetBeans`

for example, https://www.youtube.com/watch?v=_c6ft-miHr4

actually, it's not about rule, just code style, and we can set whatever style we want together (just at least, don't break the rule from PSR-2). :D

don't you think above code is more readable?